check_diff/tests/bash_commands.rs

```diff
@@ -0,0 +1,34 @@
+use check_diff::change_directory_to_path;
+use std::env;
+use std::ffi::OsStr;
+use tempfile::Builder;
+use tracing::error;
+
+#[test]
+fn cd_test() {
+    // create directory in current directory
+    let dir = Builder::new().tempdir_in("");
+
+    match dir {
+        Ok(d) => {
+            let dest_path = d.path();
+            let _ = change_directory_to_path(dest_path);
+            assert_eq!(env::current_dir().is_ok(), true);
```

Comment:
I feel like it would be a more direct test if you asserted that the current working directory is the same as the `dest_path`